src/uint/boxed.rs

```diff
@@ -395,17 +398,75 @@ impl fmt::UpperHex for BoxedUint {
     }
 }
 
+#[cfg(feature = "serde")]
+impl<'de> Deserialize<'de> for BoxedUint
+// TODO
+//where
+//    BoxedUint: Encoding,
```

Comment:
TODO: need to find out where it tags the Encoding impl as it seems to be associated directly in Self ? :exploding_head: